presto-main/src/main/java/io/prestosql/cost/ComparisonStatsCalculator.java

```diff
@@ -166,7 +167,8 @@ public static PlanNodeStatsEstimate estimateExpressionToExpressionComparison(
             case GREATER_THAN:
             case GREATER_THAN_OR_EQUAL:
             case IS_DISTINCT_FROM:
-                return PlanNodeStatsEstimate.unknown();
+                // TODO: provide better formula here
+                return inputStatistics.mapOutputRowCount(rowCount -> rowCount * UNKNOWN_FILTER_COEFFICIENT);
```

Comment:
I do believe we should apply UNKNOWN_FILTER_COEFFICIENT to all Filters we cannot otherwise estimate.
But this should be applied on FilterRule level, not on individual comparison expression level.
For example, one can have multiple such unestimated comparisons, or other expression components that we could estimate. `io.prestosql.cost.FilterStatsCalculator.FilterExpressionStatsCalculatingVisitor#estimateLogicalAnd` has special support for case when one branch could not be estimated.

Reply:
btw in the cmt msg

- replace "UNKNOWN_STATS" with "UNKNOWN_FILTER_COEFFICIENT"
- cmt title is too long

Reply:
Maybe we should default to `optimizer.default-filter-factor-enabled=true` and also support it for join (or just `FilterStatsCalculator`)?

Reply:
oh, right, there is this flag. Yes, i believe `optimizer.default-filter-factor-enabled` should be on by default. IMO for Filters only, not joins (0.9*cross is likely way off)

Reply:
> Yes, i believe `optimizer.default-filter-factor-enabled` should be on by default.

Maybe we do, but shouldn't we be conservative with CBO so that it doesn't make any really stupid choices (granted that user queries are not externally inefficient).